app/routes/admin/email/components/EmailUserEditor.js

```diff
@@ -0,0 +1,78 @@
+//@flow
+
+import React from 'react';
+import Button from 'app/components/Button';
+import { createValidator, required } from 'app/utils/validation';
+import { reduxForm } from 'redux-form';
+import { TextInput, SelectInput, CheckBox } from 'app/components/Form';
+import { Form, Field } from 'redux-form';
+
+export type Props = {
+  emailUserId?: number,
+  submitting: boolean,
+  handleSubmit: Function => void,
+  push: string => void,
+  mutateFunction: Object => Promise<*>
+};
+
+const EmailUserEditor = ({
+  emailUserId,
+  mutateFunction,
+  submitting,
+  push,
+  handleSubmit
+}: Props) => {
+  const onSubmit = data => {
+    mutateFunction({
+      id: data.id,
+      user: data.user.value,
+      internalEmail: data.internalEmail,
+      internalEmailEnabled: data.internalEmailEnabled || false
```

Comment:
why the `|| false`? if you really need a boolean you can do `!!data.internalEmailEnabled`